articles/azure-app-configuration/enable-dynamic-configuration-aspnet-core.md

```diff
@@ -130,16 +126,14 @@ A *sentinel key* is a special key used to signal when configuration has changed.
     ```
     ---
 
-    The `ConfigureRefresh` method is used to specify the settings used to update the configuration data with the App Configuration store when a refresh operation is triggered. The `refreshAll` parameter to the `Register` method indicates that all configuration values should be refreshed if the sentinel key changes.
-
-    Also, the `SetCacheExpiration` method overrides the default cache expiration time of 30 seconds, specifying a time of 5 minutes instead. This reduces the number of requests made to App Configuration.
+    In the `ConfigureRefresh` method, you register the keys in your App Configuration store to be monitored for change detection. The `refreshAll` parameter to the `Register` method indicates that all configuration values should be refreshed if the registered key changes. The `SetCacheExpiration` method specifies the minimum time, before which a new request will be made to App Configuration to check for any configuration changes. In this example, you override the default expiration time of 30 seconds specifying a time of 5 minutes instead. This reduces the potential number of requests made to your App Configuration store.
```

Comment:
>The `SetCacheExpiration` method specifies the minimum time, before which a new request will be made to App Configuration to check for any configuration changes. [](http://example.com/codeflow?start=273&length=160)

The `SetCacheExpiration` method specifies the minimum time that must elapse before a new request is made to App Configuration to check for any configuration changes.

Reply:
Sounds good. Updated.